.pre-commit-config.yaml

```diff
@@ -17,21 +17,20 @@ repos:
         # Don't commit to main branch.
     -   id: no-commit-to-branch
 -   repo: https://github.com/psf/black
-    rev: '21.6b0'
+    rev: "21.6b0"
     hooks:
     -   id: black
         # Force black to run on whole repo, using settings from pyproject.toml
         pass_filenames: false
-        args: [--config=./pyproject.toml, .]
+        args: ["--config=./pyproject.toml", "--check", "docs", "lib", "noxfile.py", "setup.py"]
```

Comment:
`tools` dir as well?

Reply:
For consistency this should be transferred to `pyproject.toml`, [I believe using using the `include` key](https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html?highlight=include#configuration-via-a-file)?

Reply:
Cool, I'll take a peek

Reply:
hmmm this is slightly frustrating, in that `include` doesn't quite behave the same as `src_paths` from `isort`... so we need to be explicit. `include` is like a filter of files to include from what's found to the call to `black`, rather than specifically only include these files and directories.

Reply:
That is frustrating, yeah.

Given we are forced to use CLI for `black` and `flake8`, I'd prefer that we use the CLI for `isort` too. Makes for less confusion for the user.